### DISCUSSION TITLE
Start up fails

### DISCUSSION THREAD

**Discussion by DaviidM** [kaorahi/lizgoban#81] [General]
Has anybody else experienced LizGoBan ceasing to start on Windows 11?

**Comment by kaorahi** [comment]
Would you give me more information? I do not have a Windows machine.

**Reply by DaviidM** [reply]
I've posted a video on YouTube (https://youtu.be/n6vnJIcwpa0) showing the problem.  In the process I found a Windows error message about a "BITS" failure.  I have no idea what BITS is or does.  I can tell you that Lizzie, Sabaki, Q5Go (analysis and play against the machine), Katrain and ogatak all work.  Obviously I have KataGo and I'm running version 11.

**Reply by DaviidM** [reply]
BITS seems to be something that limits concurrent applications on Windows Servers.  I'm not running a server operating system... so I ran a health check (dism /Online /Cleanup-Image /RestoreHealth) and a systems file check (sfc /scannow) to ensure there were no problems with the operating system.  The problem persists!

**Reply by kaorahi** [reply]
thx for the reply. You seem much more experienced in Windows than I. What I can say are...

* I got at least one report that `LizGoban 0.7.0.exe` in LizGoban-0.7.0_win_220521a.zip runs on win11 with no problem.
* lizgoban never accesses the network except when you paste (or drag & drop) a URL on it. You must see the dialog `Open https://foo/bar.sgf [Ok] [Cancel]` before the network access in this case.
* `LizGoban 0.7.0.exe` is a portable binary that runs by itself without additional files. It is actually a package of LizGoban.exe, katago-*.exe, katanetwork.gz, *.dll, etc. that are expanded at runtime. So it takes long time at startup and needs large resources. This point is different from *.exe of other GUIs.

**Reply by DaviidM** [reply]
Thanks for the quick reply.  I have found the problem and can run LizGoBan.  The issue is/was that I changed the windows settings to allow for screen magnification (my wife is visually impaired and that helps her).  This in turn created a file in my user profile - .accessibility.properties - the two lines in the file need to be prefixed with a # to make them notes.  The magnification setting need to be turned off.  With both of these in place LizGoBan works.  There is a small issue - somewhere in the depths of Windows, it decides to reinstate the two lines :) so you need to stay awake, expecially if the machine is rebooted.

**Reply by DaviidM** [reply]
BTW - this also impacts GoWrite - it will not start with unless the accessibility.properties are # out.  Not  that it is a problem for you.

**Reply by kaorahi** [reply]
Thanks for your informative report. It will help other users!

**Reply by DaviidM** [reply]
No problem.  Thank you for the software and all of your work.  Much appreciated.
